wemake_python_styleguide/violations/consistency.py

```diff
@@ -51,6 +51,7 @@
 .. autoclass:: FormattedStringViolation
 .. autoclass:: RequiredBaseClassViolation
 .. autoclass:: MultipleIfsInComprehensionViolation
+.. autoclass:: ComparisonOrderViolation
```

Comment:
Please, also add `ComparisonOrderViolation` to the `Summary` section.
Here: https://github.com/wemake-services/wemake-python-styleguide/blob/f382eb42552ca50053aeb3ff8b97c3b09b2a37e8/wemake_python_styleguide/violations/consistency.py#L28-L41